src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -157,6 +159,20 @@ const ResearchProfile = () => {
             </Typography>
           </Box>
         )}
+        {(!!personBackground || personKeywords.length > 0) && (
+          <Box sx={{ mt: '1rem', display: 'flex', flexDirection: 'column', gap: '0.5rem', width: '70%' }}>
+            <Typography variant="h3" gutterBottom>
+              {t('my_page.my_profile.field_and_background.field_and_background')}
+            </Typography>
+            <Box sx={{ display: 'flex', gap: '0.5rem', mb: '0.5rem' }}>
+              {personKeywords.length > 0 &&
```

Comment:
legg denne sjekken rundt `<Box>` i stedet, for å unngå tom div?

Reply:
husk å sjekk at det ser greit ut for brukere som mangler bakgrunn og/eller nøkkelord også

Reply:
Synes det ser greit ut nå :+1: